### PR TITLE
Never show legacy sharing options (in Finish dialog) for App Lab

### PR DESCRIPTION
**Pull request by joshlory**
Request from @poorvasingal.  Follow up to PR https://github.com/code-dot-org/code-dot-org/pull/19332.

Before:

image

After:

image

### CODE REVIEW

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -937,7 +937,7 @@ var displayFeedback = function () {
       executionError: Applab.executionError,
       response: Applab.response,
       level: level,
-      showingSharing: level.freePlay,
+      showingSharing: false,
```
Comment (Bjvanminnen) [inline apps/src/applab/applab.js:940]: Did we previously (before your recent PR) just not use showingSharing for anything in App Lab?

Comment (joshlory) [reply apps/src/applab/applab.js:940]: I'm not sure...

@poorvasingal, confirm that we *never* want to show sharing options in the "Finish" dialog for any App Lab level?

Comment (ryansloan) [reply apps/src/applab/applab.js:940]: @joshlory @poorvasingal confirmed that is correct - no share in "Finish" for any App Lab or Game Lab

Comment (joshlory) [reply apps/src/applab/applab.js:940]: Note that this change will only affect App Lab.  Per https://codedotorg.slack.com/archives/C0T0GC9GE/p1515023020000162 this doesn't currently repro in Game Lab.  We could take a preventative fix for Game Lab here if you want?

https://github.com/code-dot-org/code-dot-org/blob/2ae2985b86384eed0cd233584aa7402a671efa82/apps/src/gamelab/GameLab.js#L1118

**Pull request merged**